api/HTMLAnchorElement.json

```diff
@@ -253,10 +253,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": false
```

Comment:
There is no `media` attribute on `HTMLAnchorElement` or the `HTMLHyperlinkElementUtils` mixin in the spec. I don't think there ever has been. Same for `HTMLAreaElement`. These entries should be removed instead.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/7538 to do that.